openedx_webhooks/info.py

```diff
@@ -152,6 +153,7 @@ def is_committer_pull_request(pull_request: PrDict) -> bool:
 
     repo = pull_request["base"]["repo"]["full_name"]
     org = repo.partition("/")[0]
+    base_branch = pull_request["base"]["ref"].split('/')[0]
```

Comment:
I guess I would still prefer a pattern to match, than to hard-code the idea that the part before the slash is significant.

Reply:
FINE lol so sorry I'm being lazy about this :) try now